docs/blog/posts/2025/a-new-journey.md

```diff
@@ -1,10 +1,37 @@
 ---
-date: 2025-02-15
+date: 2025-03-01
 authors:
+    - jwurtz
     - rogerluo
     - kaihsin
     - weinbe58
     - johnzl-777
 ---
+# A new journey for Bloqade
 
-# A New Journey of Bloqade
+In 2023 we were excited to introduce bloqade, a python SDK for programming and interfacing with analog mode neutral atom hardware. Today, we introduce the next generation of Bloqade: as well as programming analog-mode computation, our new bloqade-circuits module enables programming digital and gate-based computation, with an eye on near-term NISQ demonstrations and intermediate-term fault tolerant solutions. Don’t worry; all of your favorite features of the previous generation of bloqade are still there under the `bloqade.analog` namespace, but now you can explore digital-mode computation specialized to reconfigurable neutral atoms.
+Why have we built this new module? There are plenty of incredible quantum programming packages, such as [qiskit]( https://www.ibm.com/quantum/qiskit) and [cirq]( https://quantumai.google/cirq), as well as an entire ecosystem of middleware providers with specialized pipelines to turn abstract problems into circuits. However, these packages may not be everything that is needed for efficient hardware execution on neutral atom hardware: **a circuits-only representation of quantum executions may be the incorrect abstraction for effective hardware-level programs**. This is a challenge: we want to enable everyone to maximally leverage the power of neutral atom quantum computers beyond abstract circuit representations. For this reason, we are building Bloqade to be a hardware-oriented SDK to represent hybrid executions on reconfigurable neutral atom hardware. In this way, bloqade can be integrated into the larger ecosystem—for example, codegen of qasm from a bloqade-circuits program, but be an SDK specialized to our hardware: **THE SDK for neutral atoms**.
```

Comment:
should this be a new paragraph? we should also mention PennyLane and CUDA Q if we are working with them :-)

> **a circuits-only representation of quantum executions may be the incorrect abstraction for effective hardware-level programs**.

would "incorrect" be too much here? maybe something like insufficient etc.

Other part looks good to me.